tableau/v_tableau.hs_community_service.sql

```diff
@@ -31,5 +32,7 @@ LEFT JOIN gabby.deanslist.behavior b
   ON co.student_number = b.student_school_id
  AND co.[db_name] = b.[db_name]
  AND b.behavior_category = 'Community Service'
-WHERE co.academic_year = gabby.utilities.GLOBAL_ACADEMIC_YEAR()
+ AND b.behavior_date BETWEEN co.entrydate AND co.exitdate
+WHERE co.academic_year >= gabby.utilities.GLOBAL_ACADEMIC_YEAR() - 5
```

Comment:
you can drop this if you're relying on enrollment status

Reply:
you probably also need to add rn_year = 1 so you don't get multiple rows per year